targets/TARGET_ublox/TARGET_R5030/device/TOOLCHAIN_ARM_STD/r5030.sct

```diff
@@ -22,16 +22,16 @@
  *************************************************************
  */
 
-LR_IROM1 0x00000000 0x00040000  {    ; load region size_region
+; R5030_APPS_PROC: 128 KB FLASH (0x200000) + 1024 KB SRAM (0x100000)
+LR_IROM1 0x00000000 0x20000  {    ; load region size_region
   ER_IROM1 0x00000000 0x00040000  {  ; load address = execution address
```

Comment:
Which memory area is this?

Reply:
ER_ROM1 is the first execution region. However, its size is same as of load region in this case. Corrected to 0x20000.